pkg/iac-providers/kustomize/v3/load-dir_test.go

```diff
@@ -4,13 +4,16 @@ import (
 	"fmt"
 	"os"
 	"reflect"
+	"strings"
 	"syscall"
 	"testing"
 
 	"github.com/accurics/terrascan/pkg/iac-providers/output"
 	"github.com/accurics/terrascan/pkg/utils"
 )
 
+const kustomizeErrPrefix = "error from kustomization."
```

Comment:
can this be an `error` variable?

Reply:
it isnt exacty an error message.. but a prefix